pacapt

```diff
@@ -3,7 +3,7 @@
 # Purpose: A wrapper for all Unix package managers
 # License: Fair license (http://www.opensource.org/licenses/fair)
 # Source : http://github.com/icy/pacapt/
-# Version: 2.4.3
+# Version: 2020.05.12
```

Comment:
Thanks for your contribution.

The current version/release process is manual. On the same day, there may be some minor fixes (e.g, 2.4.3, 2.4.4). This is not easy to work with `YYYY.mm.dd` version format.

Moreoever, changing the stable script (`pacapt`) will trigger build system on different systems (e,g Homebrew Formula https://github.com/Homebrew/homebrew-core/blob/master/Formula/pacapt.rb ). If possible we may avoid that.

That's said, I'm ok with new change, however, for the stable script I don't like to touch them.

What do you think?